app/models/training_period.rb

```diff
@@ -55,6 +58,11 @@ def one_id_of_trainee_present
     errors.add(:base, "Only one id of trainee required. Two given") if ids.all?
   end
 
+  def one_partnership_present
```

Comment:
In discussions and the writeup, although we did say it's either phase one (EOI) or phase 2 (fully fledged school partnership) do we really need to clear the expression of interest, if we keep it just for the record, would it be an issue 🤔 ? I guess it would cause confusion with a circular links, but is there a way we can keep both to preserve the history of an EOI? happy to discuss

Reply:
discussed with @ethax-ross @leandroalemao and then later @pete, this might be something we can do (and keep both records) we will discuss in the team meeting tomorrow before confirming way forward

Reply:
> discussed with @ethax-ross @leandroalemao and then later @pete, this might be something we can do (and keep both records) we will discuss in the team meeting tomorrow before confirming way forward

I don't think I've discussed a Rails project for at least a year.